CMakeLists.txt

```diff
@@ -78,7 +78,8 @@ if(NOT spdlog_FOUND OR "${spdlog_VERSION}" VERSION_LESS 1.5.0)
   build_spdlog()
 endif()
 
-# TODO(blast545): This section and package.xml should be updated once copyright auto is solved. See: https://github.com/ament/ament_lint/issues/237
+# TODO(blast545): This section and package.xml should be updated once copyright auto is solved.
+# See: https://github.com/ament/ament_lint/issues/237
```

Comment:
This comment looks like it has been resolved and can be removed instead.

It was added in #11 and the disabled test was re-enabled in https://github.com/ros2/spdlog_vendor/pull/12